music_assistant/server/providers/deezer/__init__.py

```diff
@@ -664,6 +664,7 @@ def parse_track(self, track: deezer.Track, user_country: str, position: int = 0)
             metadata=self.parse_metadata_track(track=track),
             track_number=position,
             position=position,
+            disc_number=getattr(track, "disk_number", 1),
```

Comment:
is it really disk number (with a k) and not disc number ?

Reply:
Yup, i wondered as well :thinking: https://deezer-python.readthedocs.io/en/stable/api_reference/resources/track.html#deezer.Track.disk_number at least according to the code and the docs. But could be a typo in the original library. Let me double check

Reply:
Nope, thats also how it is written in the official deezer api documentation: https://developers.deezer.com/api/track

Reply:
ok, good that you double checked, all good then!